internal/ioutil/print.go

```diff
@@ -1,25 +1,32 @@
 package ioutil
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"text/tabwriter"
 
+	"github.com/notaryproject/notation-go"
 	"github.com/notaryproject/notation-go/config"
-	"github.com/notaryproject/notation-go/plugin/manager"
-	"github.com/notaryproject/notation-go/verification"
+	"github.com/notaryproject/notation-go/plugin"
+	"github.com/notaryproject/notation-go/plugin/proto"
 )
 
 func newTabWriter(w io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(w, 0, 0, 3, ' ', 0)
 }
 
-func PrintPlugins(w io.Writer, v []*manager.Plugin) error {
+func PrintPlugins(ctx context.Context, w io.Writer, v []plugin.Plugin, errors []error) error {
 	tw := newTabWriter(w)
 	fmt.Fprintln(tw, "NAME\tDESCRIPTION\tVERSION\tCAPABILITIES\tERROR\t")
-	for _, p := range v {
+	for ind, p := range v {
+		req := &proto.GetMetadataRequest{}
+		metadata, err := p.GetMetadata(ctx, req)
+		if err != nil {
+			return err
```

Comment:
We should also print out this error.

Reply:
I think we should print the error in line 29 to print it in the table.
fmt.Fprintf(tw, "%s\t%s\t%s\t%v\t%v\t\n", metadata.Name, metadata.Description, metadata.Version, metadata.Capabilities, errors[ind])